html-report/index.html

```diff
@@ -14,5 +14,6 @@
     <div id="root"></div>
     <script type="text/javascript" src="${relative_path}build/app.min.js"></script>
     ${log}
+    <div class="copy content">Powered by Juno</div>
```

Comment:
And as we wanted in #36, we can add date here: 

"Generated with ❤️ by Juno at 00:56 UTC 7 June 2017"
`Generated with ❤️ by Juno ${date}`?